metrics/ci_integrations/docs/02_ci_integration_user_guide.md

```diff
@@ -134,9 +135,73 @@ destination:
 
 _**Note**: The order of source and destination parts doesn't matter - the first part can be `destination` and the second can be `source`. But both parts are required and must be presented in the configuration part._
 
-Well done! Your configuration file is ready to use within the `ci_integrations sync` command. The [next section](#making-things-work) describes
- how
- to make all things work together!
+Well done! Your configuration file is ready to use within the `ci_integrations sync` command. The [next section](#validating-configuration-file) describes how to validate the created configuration file.
+
+### Validating configuration file
+
+The CI Integrations tool provides functionality to validate the configuration files to ensure that all fields of the configuration file are correct. To do that, use the `validate` command of the CI Integrations tool. Consider the following example that demonstrates how to use the `validate` command:
+
+``` bash
+ci_integrations validate --config-file="path/to/config_file.yaml"
+```
+
+_**Note**: This step is optional. You can use the `sync` command without the configuration file validation, however, the `sync` command does not provide the detailed failure reasons, so it would be hard to figure out what went wrong._
```

Comment:
```suggestion
_**Note**: This step is optional. You can use the `sync` command without the configuration file validation. However, the `sync` command does not provide detailed failure reasons, so it may be hard to figure out what went wrong._
```